env.js

```diff
@@ -71,12 +71,60 @@ module.exports = (function () {
   };
 
   /**
-   * A JSON object that describes where to store intermediate files (data files to be processed)
-   * Known types are 'local' and 'sandcastle'.
+   * Credentials for AWS.  Only required if env.storage.default is 'aws/s3'.
    *
-   * 'local' is the default and just stores files locally
+   * The AWS_CREDENTIALS value should be a JSON string hash containing the
+   * properties 'accessKeyId' and 'secretAccessKey'.
+   *
+   * An example of a complete AWS_CREDENTIALS value:
+   *
+   * '{"accessKeyId": "<your-access-key-id>", "secretAccessKey": "<your-secret-access-key>"}'
+   */
+  env.awsCredentials = JSON.parse(config.fromEnvironment('AWS_CREDENTIALS', '{}'));
```

Comment:
I think the best strategy is to designate an environment-specific IAM user for each service (e.g. jellyfish-prod).  The credentials would be for this user.  This user would get tightly controlled access to just the resources that the service needed on that cluster.  So, for example, the jellyfish-prod user would have GetObject/PutObject access to the tidepool-data-raw-prod bucket only.  If we needed to add another resource (say, AWS RedShift), we authorize the jellyfish-prod user, not create another user for RedShift.

So, that's a long-winded way of saying that I think there should only be one AWS user per service per cluster.  Therefore, there is only one set of credentials.  So, I don't think name spacing the credentials is necessary.

Do you agree?